src/main/flight/pid.c

```diff
@@ -487,6 +486,10 @@ STATIC_UNIT_TESTED FAST_CODE_NOINLINE float pidLevel(int axis, const pidProfile_
 #ifdef USE_GPS_RESCUE
     angleTarget += gpsRescueAngle[axis] / 100.0f; // Angle is in centidegrees, stepped on roll at 10Hz but not on pitch
 #endif
+#ifdef USE_POS_HOLD_MODE
+    angleTarget += autopilotAngle[axis]; // autopilotAngle is in degrees, stepped at 10Hz
+#endif
```

Comment:
Do we need to check if either mode is active ?

Reply:
No, because currently anytime the flight mode gps_rescue_mode is active, it over-rides position hold.

Reply:
using define on it self does not tell which mode is active so it most be override elsewhere ?

Reply:
Yes, in core.c, there is a check.

Reply:
When both defines are true it will always execute line 490 here.

Reply:
Yes this is true.  But in core.c you will see that the activation of GPS Rescue flight mode will immediately suppress Position Hold.  The two cannot coexist.  GPS Rescue takes precedence.  That's why only one of these two factors will ever be non-zero.